### PR TITLE
MARP-1333 Missing setup path for README_DE.md

### PR DESCRIPTION
**Pull request by quanpham-axonivy**
Included README_DE.md in pom.xml and zip.xml

### CODE REVIEW

File: deepl-connector-product/zip.xml [hunk 1]
```diff
@@ -20,6 +20,7 @@
       <outputDirectory>/</outputDirectory>
       <includes>
         <include>README.md</include>
+                <include>README_DE.md</include>
```
Comment (ivy-rew) [inline deepl-connector-product/zip.xml:23]: please use the same intend as the existing pom.xml, to keep the formatting in harmony

File: deepl-connector-product/pom.xml [hunk 2]
```diff
@@ -46,6 +46,8 @@
                 <copy file="README.md" tofile="${project.build.directory}/README.md" />
                 <loadfile property="variables.yaml" srcFile="${variables.yaml.file}" encoding="UTF-8" />
                 <replace file="${project.build.directory}/README.md" token="@variables.yaml@" value="${variables.yaml}" />
+                <copy file="README_DE.md" tofile="${project.build.directory}/README_DE.md"/>
+                <replace file="${project.build.directory}/README_DE.md" token="@variables.yaml@" value="${variables.yaml}"/>
```
Comment (ivy-rew) [inline deepl-connector-product/pom.xml:50]: :heavy_check_mark:

**Review by ivy-rew — APPROVED** [review]
nice, thanks for the fix

**Review by ivy-rew — APPROVED** [review]

**Review by vhhoang-axonivy — APPROVED** [review]
LGTM

**Pull request merged**